packages/cozy-intent/src/api/models/methods.ts

```diff
@@ -1,6 +1,6 @@
 import { AppManifest, FlagshipUI } from '../../api'
 
-export type NativeMethodsRegister = {
+export interface NativeMethodsRegister {
```

Comment:
Did you encounter some limitations with the previous `type` notation?

Reply:
Actually I didn't even notice that change. It seems it was made automatically with the new config I added to cozy eslint config for TS files: `'plugin:@typescript-eslint/strict'`